packages/@react-aria/selection/src/useSelectableItem.ts

```diff
@@ -156,7 +165,8 @@ export function useSelectableItem(options: SelectableItemOptions): SelectableIte
   if (shouldSelectOnPressUp) {
     itemPressProps.onPressStart = (e) => {
       modality.current = e.pointerType;
-      if (e.pointerType === 'keyboard') {
+      longPressEnabledOnPressStart.current = longPressEnabled;
+      if (e.pointerType === 'keyboard' && (!hasAction || isSelectionKey())) {
```

Comment:
I think we're doing this because onPressStart doesn't return the full event? so we can't check the key/code on e at this point in the code, instead we rely on the event the browser is currently processing through window.event?

Reply:
Yeah...